src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -602,32 +602,71 @@ private TypeParameterConstraintKind GetConstraintKinds()
         }
     }
 
-    internal sealed class SourceMethodTypeParameterSymbol : SourceTypeParameterSymbolBase
+    internal abstract class SourceMethodTypeParameterSymbol : SourceTypeParameterSymbol
```

Comment:
>abstract class SourceMethodTypeParameterSymbol [](http://example.com/codeflow?start=13&length=46)

What does this refactoring enable?

Reply:
> What does this refactoring enable?

All implementations of method type parameters in source have a common base distinct from the base of type type parameters